### ISSUE TITLE
Possible bug: IP exception only seems to work on one machine connected to network.

### ISSUE THREAD

**Issue by jcgallaher** [jcgallaher/website-maintenance-mode#3]
Upon doing more testing. The redirect is working, but the IP address exception only seems to work on the machine I'm making the changes with, but other computers on the network with the same external address get the maintenance page. For example, my wife's computer, my iPhone, and my daughter's iPhone show the maintenance page. I've cleared the cache and retested it on every device to ensure that the browser cache isn't affecting the results, but that doesn't seem to have any influence. I can confirm this since on my machine, I changed the IP address just one number, saved it, restarted Apache, refresh the page, and the maintenance page shows up again. Change it back, to the correct IP, save it, restart Apache, refresh the page, and I see the site as normal.

I'm not sure if this is a bug or just the behavior of Apache. If anyone can let me know if they're getting the same results that I do, that would be much appreciated!

**Comment by jcgallaher** [comment]
I discovered that the root cause of the issue has to do with the way IPv6 works when you're behind a router. I confirmed this by checking my router's System Log - IPv6 Network Information. Each computer on the network had a different IPv6 address. I also tested it by changing the IP address for that other machine and it showed the normal site.

If additional people are working on the site, they need to check their IP too and it needs to be added into RewriteCond %{REMOTE_ADDR} too.

I'll be looking into a solution for multiple IP's and update the documentation accordingly.
